modules/n1ql/pages/n1ql-language-reference/insert.adoc

```diff
@@ -1297,20 +1305,19 @@ EXPLAIN INSERT INTO `travel-sample` (key UUID(), value _airport)
                       "exact": true,
                       "range": [
                         {
-                          "high": "\"Heathrow\"",
+                          "high": "\"airport\"",
                           "inclusion": 3,
-                          "low": "\"Heathrow\""
+                          "low": "\"airport\""
                         }
                       ]
                     }
                   ],
                   "using": "gsi"
                 },
                 {
-                  "#operator": "IndexScan3", // <1>
-                  "as": "_airport",
-                  "index": "def_type",
-                  "index_id": "4877ed06167c9af8",
+                  "#operator": "IndexScan3",
```

Comment:
Make sure to add the callout numbers back into the examples when you update them
```suggestion
                  "#operator": "IndexScan3", // <1>
```